frontend/src/components/mine/Tailings/FileUpload.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { FilePond, File, registerPlugin } from "react-filepond";
+import "filepond/dist/filepond.min.css";
```

Comment:
I don't think we should be referencing anything in our `/dist` in our `/src` files. This would either go into a `vendors` directory or wouldn't need to be imported at all (since the module `react-filepond` should include all of its dependence source files.

Reply:
The example code for filepond includes importing the .css styles separately like this. I'm not sure what you are suggesting is the alternative?

Reply:
Apologies. I misread and thought this was coming from our own `/dist` dir (an output from our build system). I reread it after your response and see what's actually happening.

Importing a CSS file like that still seems odd to me, but I checked the docs and didn't immediately see a reasonable alternative. Will mark as resolved